DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -648,5 +648,28 @@ public void Should_have_null_next_section_id_if_there_is_no_next_section()
             // Then
             sectionContentViewModel.NextSectionId.Should().BeNull();
         }
+
+        [Test]
+        public void Percent_complete_should_correctly_be_floored()
```

Comment:
There are already tests about this further up in this file I believe. This test should be grouped with them 🙂